docs/user-guide/how_to_use_numa_aware.md

```diff
@@ -0,0 +1,158 @@
+# NUMA Aware User Guide
+
+## Environment setup
+
+### Pre-Condition
+
+```
+1. Enable cpu manager and set policy to "static"

```

Comment:
Just curious, is setting cpu manager policy to "static" necessary ?

Reply:
Because the cpu manager on kubelet handle  the cpu assign action  and  the shceduler just determines which node can be scheduled , but does not determine cpu allocation.
So  for exclusiving  CPUs , set cpu manager policy to "static"

Reply:
Yes, l know what you mean.
My question is l think topology manager will still work if cpu manager policy is not static(don't want exclusiving CPUs), is it right?

Reply:
topology manager will still work for others resources，but CPU resource Limits  just processes  by CFS in linux kerne if the cpu manager policy is  None.

Reply:
So `Enable cpu manager and set policy to "static"` is optional for numa awareness. It’s just that in some cpu-intensive scenarios, it’s better to set cpu manager policy to static.